### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Sleptco**
- [DEMO LINK](https://Slepco.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Slepco.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,137 @@
...
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="margin-buttom">
+        <legend>Personal information</legend>
+        <label class="group">
+          Surname:
+          <input
+          type="text"
+          name="surname"
+          autocomplete="off"
+          >
```
Comment (alexandra-protyanova) [inline src/index.html:23]:
```suggestion
          <input
            type="text"
            name="surname"
            autocomplete="off"
          >
```

Comment (alexandra-protyanova) [reply src/index.html:23]: read checklist and fix code style everywhere

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,137 @@
...
+      <fieldset class="margin-buttom">
+        <legend>An interesting fact about you!</legend>
+        <label class="group">
+          Do you love cats?
+          <input
+          type="radio"
+          name="cats"
+          value="yes"
+          >
+          Yes
```
Comment (alexandra-protyanova) [inline src/index.html:88]: yes and no should have separate label for them

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,141 @@
...
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="group">
+          Name:
+          <input type="text"
+            name="name"
+            autocomplete="off"
+          >
```
Comment (DarkMistyRoom) [inline src/index.html:30]:
```suggestion
          <input 
            type="text"
            name="name"
            autocomplete="off"
          >
```

Comment (DarkMistyRoom) [reply src/index.html:30]: each attribute on a new line

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,141 @@
...
+        <legend>Personal information</legend>
+        <label class="group">
+            Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="group">
```
Comment (DarkMistyRoom) [inline src/index.html:25]: Add empty lines between multiline sibling blocks of HTML

File: src/index.html [hunk 5]
```diff
@@ -11,7 +11,141 @@
...
+            name="password"
+            required
+            minlength="5"
+            maxlength="25"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="margin-buttom">
+        <legend>An interesting fact about you!</legend>
+        <label class="group">
```
Comment (DarkMistyRoom) [inline src/index.html:81]: Don't use this common label, because when you click on the question, yes becomes selected

File: src/style.css [hunk 6]
```diff
@@ -1 +1,14 @@
...
+
+.group {
+  margin-bottom: 10px;
+  display: block;
+}
+
+.group:last-child {
+  margin-bottom: 0;
+
+}
```
Comment (DarkMistyRoom) [inline src/style.css:14]:
```suggestion
  margin-bottom: 0;
}
```

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
You should add links to the description https://www.loom.com/share/da672f3d083c4247960d02d43f7d527a

And your links don't work. Did you deploy the project?

File: src/index.html (hunk 5 above)
Comment (Sleptco) [reply src/index.html:81]: image
  прибрав. не проходить тест. Як це виправити?

File: src/index.html [hunk 7]
```diff
@@ -11,7 +11,154 @@
...
+      <fieldset class="margin-buttom">
+        <legend>An interesting fact about you!</legend>
+        <div class="group">
+        Do you love cats?
+          <label>
+            <input
+            type="radio"
+            name="cats"
+            value="yes"
+            >
```
Comment (DarkMistyRoom) [inline src/index.html:95]:
```suggestion
              type="radio"
              name="cats"
              value="yes"
            >
```

Comment (DarkMistyRoom) [reply src/index.html:95]: fix it everywhere

File: src/index.html [hunk 8]
```diff
@@ -11,7 +11,154 @@
...
+            Yes
+          </label>
+          <label>
```
Comment (DarkMistyRoom) [inline src/index.html:98]:
```suggestion
          </label>
          
          <label>
```

File: src/index.html [hunk 9]
```diff
@@ -11,7 +11,154 @@
...
+            max="10"
+          >
+        </label>
+
+      </fieldset>
+      <fieldset class="margin-buttom">
+        <legend>Additional info:</legend>
+        <label class="group">
+          Comments:
+          <textarea></textarea>
```
Comment (DarkMistyRoom) [inline src/index.html:149]: Each field should have name attribute

Comment (DarkMistyRoom) [reply src/index.html:149]: and autocomplete off

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Links still don't work. Did you deploy the project after the changes? Ask in the chat, if something goes wrong

**Comment by Sleptco** [comment]
> * [DEMO LINK](https://Slepco.github.io/layout_html-form/)
> * [TEST REPORT LINK](https://Slepco.github.io/layout_html-form/report/html_report/)

**Pull request closed without merging**